### ISSUE TITLE
[v2.alpha.62] subpages are not opening at all after upgrading from alpha.61 to alpha.62

### ISSUE THREAD

**Issue by MateuszDabrowski** [facebook/docusaurus#3367]
## 🐛 Bug Report

After updating to alpha.62 I'm able to open main page of my site, but subpages are not loading at all.
After checking Network logging it seems that all files are loading correctly but the content file itself - it returns blank.

### Have you read the [Contributing Guidelines on issues](https://github.com/facebook/docusaurus/blob/master/CONTRIBUTING.md#reporting-new-issues)?

Yes

## To Reproduce

1. Update package.json to alpha.62
2. npm run start
3. After fixing https://github.com/facebook/docusaurus/issues/3366 main page is shown
4. After clicking any internal link, a blank page is shown

## Expected behavior

A page should render as on alpha.61

## Actual Behavior

Browser is loading, but the page stays blank

## Your Environment

Docusaurus version used: alpha.62
Environment name and version (VSCode Version: 1.48.2, node v14.2.0, Safari 13.1.2, Firefox DevEd 81.0b1):
Operating system and version (desktop or mobile): Macbook Pro on Catalina 10.15.6

## Reproducible Demo

https://github.com/MateuszDabrowski/mateuszdabrowski.pl/tree/772446b0b3a84c69d02cf18c290248818501151b + bump the version of docusaurus to alpha.62

**Comment by MateuszDabrowski** [comment]
Additional information:
On initial npm run start I also received error:
```
The docs plugin option homePageId=docs-and-snippets is deprecated. To make a doc the "home", prefer frontmatter: "slug: /"
The docs plugin option homePageId=ideas is deprecated. To make a doc the "home", prefer frontmatter: "slug: /"
```
that blocks compilation. Once I deleted the homePageId setting and added slug, the compilation finished.

I'm not able to check whether this fixed the issue or not, due to the above bug.

This, however, shouldn't be the reason for the above bug, as also other pages, that did not have homePageId before, are not loading.

**Comment by slorber** [comment]
@MateuszDabrowski I don't see the errors you mention here.

See my PR to your repo with alpha 62 working (dev + prod mode).

https://github.com/MateuszDabrowski/mateuszdabrowski.pl/pull/1

![image](https://user-images.githubusercontent.com/749374/91748938-131b6780-ebc1-11ea-9e82-db24b719a04b.png)

I'll fix the sidebars.json issue reported here: https://github.com/facebook/docusaurus/issues/3366

The rest works fine to me

**Comment by MateuszDabrowski** [comment]
As mentioned here https://github.com/MateuszDabrowski/mateuszdabrowski.pl/pull/1 it seems that the issue with subpages not opening might have been either bad downloads of the files during npm install or non-numbered change in files of the alpha.62 or a problem with mine machine. Today the same steps as yesterday allowed all subpages to load correctly.

**Comment by DJM0** [comment]
I had the same going from alpha.61 to alpha.64 by editing my `package.json` and doing an `npm install`.

No errors from `npm start` (although I did remove the deprecated `homePageId` option) and the home page rendered as expected, but docs would show as blank/white pages with many React related errors in the console.

After finding this issue, I found removing both `node_modules/` and `package-lock.json` before `npm install` fixed the issue.
